11-Syntax/03-Type-Level-Programming-Syntax/src/02-Basic-Syntax/01-Defining-Custom-Kinds.purs

```diff
@@ -19,11 +19,11 @@ foreign import data Value_Level_Instance2 :: Value_Level_Type_Kind
 -- despite the "foreign import" syntax!
 ----------------------
 
--- Using the value-level type Boolean as an example...
-data Boolean_ = True_Value | False_Value
+-- Using a Boolean-like value-level type as an example...
+data YesNo = Yes | No
 
-foreign import kind Boolean
+foreign import kind YesNo
 
-foreign import data True :: Boolean
-foreign import data False :: Boolean
+foreign import data Yes :: YesNo
+foreign import data No :: YesNo
```

Comment:
This no longer distinguishes the instance (e.g. `data YesNo = Yes`) from the type (e.g. `kind YesNoKind = Yes`).

Perhaps we should distinguish it like this...?
```purescript
data YesNo = Yes_Value | No_Value

foreign import kind YesNoKind
foreign import data Yes :: YesNoKind
foreign import data No :: YesNoKind
```

Reply:
How about going with
```purescript
data YesNo = Yes | No

foreign import kind YesNoKind
foreign import data YesK :: YesNoKind
foreign import data NoK :: YesNoKind
```
like I mentioned in the first comment?

Reply:
Yes, that sounds good.